theories/Compat/README.md

```diff
@@ -0,0 +1,4 @@
+When adding a file in this directory, please name it `CoqXY.v` and
+prepend `From Stdlib Require Export CoqXY.` to the previous file.
+When removing the last remaining content of some file, please also
+remove the file altogether.
```

Comment:
That's already the most important info to include in this README, but I would have hoped for a little bit more explanations about this infrastructure here.